### DISCUSSION TITLE
community/asset_library/using_assetlib

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#32] [User-contributed notes]
# community/asset_library/using_assetlib

On the website: Overview: As mentioned before, you can access the web frontend of the Asset Library on Godot's official website. This is what it looks like when you first visit it: image0 At the to...

https://docs.godotengine.org/en/latest/community/asset_library/using_assetlib.html

**Comment by goatchurchprime** [comment]
If you want to download a zip file that is not of the whole repository (for example, the distributed code is in the addons folder, and a test example is in the top level), then you need to make your own zip file in the releases.  See here https://github.com/godotengine/godot-asset-library/issues/313#issuecomment-1939430110 for instructions on how to make a github action to do it automatically, and don't forget to set the "repository provider" to "custom", even though it's still really github.

**Comment by v1993** [comment]
Is it possible to somehow override conflicting files? I can't figure out if it's even possible to update an asset to a new version using Godot's AssetLib interface. The closest thing I've found is https://github.com/godotengine/godot/issues/6277 which was closed unsolved in 2020.

**Reply by Calinou** [reply]
It doesn't appear to be possible to override conflicting files. If you click the error message and select all files, they won't override existing files in the project.

To update to a new version within the editor, you need to remove the existing `addons/<addon_name>` folder, then install the new version using the AssetLib tab.
